### ISSUE TITLE
Having monthly(?) free credits for moving 50 messages at once when there is more to move

### ISSUE THREAD

**Issue by isgursoy** [laralove143/interchannel-message-mover#21]
Currently, we have an error saying "i can work with up to 50 messages, if you need a higher limit, please join the support server and tell lara" then nothing happens when there are 50+ messages to move, and yes, the reason perfectly makes sense, API limits.

But we could do it if there are =< 50 messages. Maybe you can think of changing the way how the "move messages below" mode works without being abusive.
Either we can select up to 50 messages or better; the bot moves the next 50 messages at once, and there can be n amount of recurring credits for being able to do that.

In this case, instead of an error, we can have a warning message explaining the case:
- Information regarding remaining credits.
- Telling about Discord API limits.
- Mentioning solution; hosting own instance or switching to the premium plan.

**Comment by laralove143** [comment]
good idea, im thinking 3 monthly requests per server for this, if this turns out to cause issues (likely will as it scales up) i can implement a queue and put longer requests behind in the queue, the premium queue will be in front for people that dont want to wait, meaning it'll probably be another private bot hosted in a different server

for the premium plan i think 3 dollars monthly is a fair price? for regional pricing they can contact me, i dont really think there'd be much demand for this at least currently, they'd also get [the benefits of sponsoring me](https://github.com/sponsors/laralove143)

**Comment by isgursoy** [comment]
In my opinion, many server admins just don't know that these two are possible with those bots, but moving and editing (others) messages are two critical gaps of Discord management as of today; this is quite clear, maybe the most important two.
Needed everywhere, with everyone, forever. Especially [editing](https://github.com/laralove143/message-bender).

Until Discord decides to fix these, you are solving two big problems and no doubt that there will be demand (plus the resulting load) as more people hear about these bots.

I am not sure vote-to-premium helps here, but $3 seems fine to me to solve these two.

**Comment by laralove143** [comment]
thank you, tho i'll put the premium behind in the impl order because i dont wanna charge for an unfinished product (not an aaa gamedev)

**Comment by laralove143** [comment]
#11 will offer an option to use a credit to actually reupload messages too, since this is also very expensive
